spk/salt-minion/src/conf/privilege

```diff
@@ -2,33 +2,5 @@
 	"defaults":{
 		"run-as": "package"
 	},
```

Comment:
I recommend removing the file and let the framework generate it for you using `SERVICE_USER = auto`

Reply:
I plan to use same account for spk/salt-master and spk/salt-minion packages so that was why I did not wanted an auto user.

BTW. I noticed that the user is never deleted in /etc/passwd when you uninstall a package. A "feature" / bug ?